package.json

```diff
@@ -90,7 +90,7 @@
     "@scalableminds/prop-types": "^15.6.1",
     "@scalableminds/saxophone": "^0.5.0",
     "airbrake-js": "^0.5.8",
-    "antd": "~3.2.3",
+    "antd": "~3.5.4",
```

Comment:
Since antd didn't really follow semver (as this is a breaking change), I would suggest to pin the exact version by dropping the `~`.

Reply:
The minor version is already pinned with the ~ modifier, so only the patch level could increase. But indeed, the breaking change happened in a patch version (I just checked again), so let's pin it to the exact version ^^